style.css

```diff
@@ -1,345 +1,341 @@
-  content: ''; /* only works with this added! */
-  display: block;
-  transform: translateX(-100%); /* this is offsetting it 100% to the side */
-  transition: transform .25s ease-in-out;
-}
-
-.movie-title.strikethrough::after {
-  transform: translateX(0); /* This brings the border back to the text */
- }
-
-
-@media (max-width: 700px) {
+  
+  html{
+    overflow: hidden;
+  }
+  
+  body {
+    margin: 0;
+    font-family: 'Gochi Hand', 'Just Another Hand', cursive;
+    /* width: 100vw;  */
+    width: calc(var(--vw, 1vw) * 100);
+    height: 100vh; 
+    overflow: auto; 
+  }
+  
+  /* background signal screen */
+  
+  .no-signal {
+    width: 100%; 
+    height: 100%; 
+    position: fixed; /* changed to fixed */
+    z-index: -1;
+    background: linear-gradient(to right, var(--ocean-blue) 0%, var(--darkOcean-blue) 14.28571%, var(--darkGrey-color) 14.28571%, var(--strongShade-grey) 28.57143%, var(--bright-pink) 28.57143%, var(--sharp-retroPink) 42.85714%, var(--lightGrey-color) 42.85714%, var(--light-greyTwo) 57.14286%, var(--bright-greenMint) 57.14286%, var(--lightGreen-color) 71.42857%, var(--darkGrey-color) 71.42857%, var(--strongShade-grey) 85.71429%, var(--white-color) 85.71429%, var(--tinted-grey) 100%);
+  }
+  
+  .no-signal:before{
+      top: 0;
+      height: 68.47826%;
+      background: linear-gradient(to right, var(--white-color) 0%, var(--tinted-grey) 14.28571%, var(--sunnyYellow-color) 14.28571%, var(--mustard-yellow) 28.57143%, var(--bright-greenMint) 28.57143%, var(--lightGreen-color) 42.85714%, var(--lime-green) 42.85714%, var(--neonBright-green) 57.14286%, var(--bright-pink) 57.14286%, var(--sharp-retroPink) 71.42857%, var(--rich-red) 71.42857%, var(--venetian-red) 85.71429%, var(--ocean-blue) 85.71429%, var(--darkOcean-blue) 100%);
+  }
+  
+  .no-signal:before, .no-signal:after {
+    content: '';
+    position: absolute;
+    left: 0;
+    z-index: 1;
+    width: 100%;
+  }
+  
+  .no-signal:after {
+    bottom: 0;
+    height: 21.73913%;
+    background: linear-gradient(to right, var(--veryDark-cyan) 0%, var(--darkTeal-color) 16.66667%, var(--white-color) 16.66667%, var(--tinted-grey) 33.33333%, var(--midnight-blue) 33.33333%, var(--milkyWay-deepBlue) 50%, var(--lightGrey-color) 50%, var(--light-greyTwo) 66.66667%, var(--steel-grey) 66.66667%, var(--fossil-grey) 83.33333%, var(--darkGrey-color) 83.33333%, var(--strongShade-grey) 100%);
+    font-family: 'Gochi Hand', 'Just Another Hand', cursive;
+  }
+  
+  
+  h1 {
+    margin: 0;
+    position: fixed;
+    width: 100vw;
+    z-index: 20;
+    padding: 0.3em;
+    background-color: var(--onyx-black);
+    text-align: center;
+    color: var(--mustard-yellow);
+  }
+  
+  /* Styling for TV button */
+  
   .tv {
-   margin: 0;
-   z-index: 20;
+    margin: 3.5em 1em auto 1em;
+    position: fixed;
+    z-index: 10;
+    background-color: transparent;
+    border: none;
+    height: 10em;
   }
+  
   .tv img {
-  width: 7em;
+    width: 10em;
```

Comment:
It' seems likes all the CSS changes are just the responsive branch merging with your comp? I'm going to merge and will double check the deploy after :)